worlds/pokemon_rb/__init__.py

```diff
@@ -67,6 +67,15 @@ class PokemonWebWorld(WebWorld):
         ["Shiny"]
     )
 
+    setup_es = Tutorial(
+        setup_en.tutorial_name,
+        setup_en.description,
+        "Français",
+        "setup_fr.md",
+        "setup/fr",
+        ["TheLynk"]
+    )
+
     tutorials = [setup_en, setup_es]
 
```

Comment:
I'm not sure how to recommend changes so that you can just click a button and implement them, but I ran a local test, and this overwrote the Spanish Tutorial. The 'setup_es' Tutorial should instead probably be 'setup_fr', and setup_fr should be added to tutorials on line 79.